packages/react/src/primitives/primitives.ts

```diff
@@ -0,0 +1,40 @@
+export { Alert } from './Alert';
```

Comment:
Is this change necessary, now that our exports are all named exports?

Reply:
I don't see the relation with named exports. There will be hooks exported  which we don't want to include in primitives?

Reply:
OK, I understand this a bit better. Still feels awkward, but it makes sense.